geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SftBuilder.scala

```diff
@@ -22,10 +19,10 @@ import scala.reflect.runtime.universe.{Type => UType, _}
 
 
 class SftBuilder {
+  import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
 
-  private val entries = new ListBuffer[String]
-  private var splitterOpt: Option[Splitter] = None
-  private var enabledIndexesOpt: Option[EnabledIndexes] = None
+  val entries = new ListBuffer[String]
+  var enabledIndexesOpt: Option[EnabledIndexes] = None
```

Comment:
you could possibly make index-ing something agnostic...then the datastore could decide what to do with it